bin/diktat

```diff
@@ -18,6 +18,7 @@ IFS=$'\n'
 declare -i COLOR=0
```

Comment:
btw, didn't we look at kotlin-native as running task for `java -jar diktat`? it will allow to use `kotlinx-cli` for such requests